sdk/src/loom2d/Display/DisplayObjectContainer.ls

```diff
@@ -155,6 +155,8 @@ package loom2d.display
          *  requested, the child will be disposed right away. */
         public function removeChildAt(index:int, dispose:Boolean=false, fireEvents:Boolean = true):DisplayObject
         {
+            var numChildren:int = mChildren.length; 
```

Comment:
Does this shadow the getter of the same name? Does that resolve properly? (Local first?)